### PR TITLE
Add Coinbase Pro pair converter

### PR DESCRIPTION
**Pull request by qwhelan**
Add a new `ccxt`-based pair converter locked to `Coinbase Pro`. This requires a few changes:
 - Addition to `_EXCHANGE_DICT`
 - Override the default `_TIME_GRANULARITY` list when using `Coinbase Pro`, as it supports `6h` instead of `4h`: https://docs.cloud.coinbase.com/exchange/reference/exchangerestapi_getproductcandles
 - Upgrade `ccxt` to the minimum version with my fix for Coinbase Pro: https://github.com/ccxt/ccxt/pull/17646

### CODE REVIEW

**Review by eprbell — COMMENTED** [review]
This LGTM, but let's wait for @macanudo527's review, because he's the CCXT expert.

**Comment by macanudo527** [comment]
I'm doing a full pull now. I know there was an issue with CCXT and Binance.com. That is why we locked the version. If the pull works without issues, this should be good to go.

Sorry about the delay. We just went back to work after a major holiday here in Japan.

**Comment by macanudo527** [comment]
This looks good to me. The pull was successful,

**Comment by eprbell** [comment]
> This looks good to me. The pull was successful,

Great! @qwhelan, Github reports conflicts: after they are fixed we can merge.

**Comment by qwhelan** [comment]
@eprbell Rebased, conflict in `setup.cfg` should be resolved.

**Review by eprbell — APPROVED** [review]
Ship it!

**Pull request merged**
